package.json

```diff
@@ -26,7 +26,7 @@
     "@date-io/luxon": "^2.10.6",
     "@material-ui/core": "^4.11.0",
     "@material-ui/lab": "^4.0.0-alpha.56",
-    "@material-ui/pickers": "4.0.0-alpha.12",
+    "@material-ui/pickers": "^3.2.10",
```

Comment:
we need the alpha for my date-picker, can we revert this?

Reply:
once you do this, you'll need to rerun yarn and resolve the yarn.lock conflict

Reply:
This is resolved!